### ISSUE TITLE
Question about box features

### ISSUE THREAD

**Issue by MLRadfys** [deeplearning-wisc/vos#46]
Hi,

in your paper you explained that you use the features of the penultimate layer for VOS. You also describe that you tested an earlier layer, which didn't work as well though.

Now Iam wondering:
- In Appendix F you write the the earlier layer had 1024 features. Where exactly did you extract the features? Was it after the first FC layer? For me it seems that both the penultimate layer and the layer before have 1024 features.
- Have you also tried to just use the features of the classification head (class logits) because of the smaller dimensionality?

Thanks in advance,

kind regards,

/M

**Comment by d12306** [comment]
hi, @MLRadfys , 1) yes, after the first FC layer. 2) I have not tried to use the logits for synthesis.
